packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -90,12 +113,17 @@ export default class VTable {
                     }
                 });
             });
+            const formatInfo = getTableFormatInfo(this.table);
+            if (formatInfo) {
+                this.applyFormatAndStyle(formatInfo);
+            }
         } else if (this.table) {
             this.table.parentNode.removeChild(this.table);
         }
     }
 
     /**
+     * @deprecated

```

Comment:
If there is no conflict betwen applyFormat and applyFormatAndStyle, no need to create the new API, but just reuse the existing one. Ping me if you have any question about this part.

Reply:
And we should still accept "Partial<TableFormat>"